src/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -67,7 +68,7 @@ protected Delegate(Type target, string method)
             if (method == null)
                 throw new ArgumentNullException(nameof(method));
 
-            RuntimeType rtTarget = target as RuntimeType;
+            RuntimeType? rtTarget = target as RuntimeType;
```

Comment:
`if (!(target is RuntimeType rtTarget))`

Consistency with the other change, multiple places